lib/Conversion/TritonGPUToLLVM/PtxAsmFormat.cpp

```diff
@@ -125,7 +125,7 @@ std::string PTXBuilder::dump() const {
     lines.push_back(exec->dump());
   }
 
-  return strJoin(lines, "\r\n");
+  return strJoin(lines, "\n\t");
```

Comment:
For pretty printing

Reply:
`\r\n` is the line break in windows, the code here is written in Windows I think, simply `\n` is not enough for Windows.
`\n\t` why we need a `\t` here? Will it just indent the code from the second line?

Reply:
(1) If we use `\r\n`, then the generated PTX will show a `^M` character at `\r`

(2) The PTX generated by LLVM NVPTX has a '\t' at the beginning of each line. Without '\t', the generated code looks like this:
```
    line1;
    line2;
    line3;^M
line4;
    line5;
    line6;
```